### DISCUSSION TITLE
This is funny

### DISCUSSION THREAD

**Discussion by IMGROOT2** [For-0/vocabustudy#168] [General]
Oh dear...

![image](https://user-images.githubusercontent.com/116324098/197358369-195ec80c-eff5-440c-a264-3fa95f93ec2f.png)
![image](https://user-images.githubusercontent.com/116324098/197358372-e3df8cc8-91c6-43a3-98fb-8f93f2237ed0.png)

**Comment by grimsteel** [comment]
I was literally in google domains just now looking at Vocabustudy domains!

Because For Zero is non profit (literally in the name) we can have forzero.org I think

![image](https://user-images.githubusercontent.com/115331356/197358479-a81a9dd0-2baa-4918-bfa3-9d36bd0bdff0.png)
Wow I'm surprised it's not taken

vocabustudy.org and vocabustudy.com are also available

**Reply by PrakmO** [reply]
I think we should do vocabustudy.org

**Reply by grimsteel** [reply]
We should

**Reply by IMGROOT2** [reply]
Funding? Can we be sponsored by RRISD or something

**Reply by grimsteel** [reply]
We just need $12 a year...
@DG3QRT1's going to turn his locker into a donation box (is that allowed?)

**Reply by IMGROOT2** [reply]
> We just need $12 a year...

ik lol but the thing is that if you launch the app to the public we'll have to pay for more storage, etc.

we could literally set up a gofundme and... advertise? 

> @DG3QRT1's going to turn his locker into a donation box (is that allowed?)

hopefully! idk tbh

**Reply by grimsteel** [reply]
> if you launch the app to the public 

Isn't is already launched to the "public"?

> gofundme

lol is that another one of the countless funding things? There's also buymeacoffee or opencollective or just paypal if you want to stay simple

Also Firebase is pretty cheap honestly

**Reply by IMGROOT2** [reply]
For me, "Public" means that you've got at least 500 users, and people in the world actually know what it is and that it's passing around by word of mouth

**Reply by IMGROOT2** [reply]
> lol is that another one of the countless funding things? There's also buymeacoffee or opencollective or just paypal if you want to stay simple

LOL yeah it is but buymeacoffee works too

**Comment by grimsteel** [comment]
![image](https://user-images.githubusercontent.com/115331356/201494162-62f8058b-c9c7-4da4-84c7-8aeaaf627da9.png)

Guess who

**Reply by grimsteel** [reply]
The SSL certificate will take some time (a couple hours) to propagate

**Reply by IMGROOT2** [reply]
WOAH WHAT?!?!?! who gave you the money? congratulations! uhh.. woah!

**Reply by IMGROOT2** [reply]
did you buy through google, or some other platform?

**Reply by IMGROOT2** [reply]
IT WORKS!!!

**Reply by IMGROOT2** [reply]
vocabustudy.org works!
can i send an announcement?

**Reply by IMGROOT2** [reply]
oop 
![image](https://user-images.githubusercontent.com/116324098/201494336-2ccdb096-cb80-474f-bf53-b7d474b3c9ee.png)
lol

**Reply by grimsteel** [reply]
I bought it through google - there was transparent pricing and it auto verified with search console and firebase!

**Reply by grimsteel** [reply]
Yes I will need to add it to all the domain allowlists... if I can find them

**Reply by IMGROOT2** [reply]
> I bought it through google - there was transparent pricing and it auto verified with search console and firebase!

perfect! the only thing is that if you had bought through GoDaddy, i could have gotten a free domain email, since Microsoft 365 Family gives two of them for free i think... but that's honestly useless anyway, and GoDaddy might be more expensive and take up more time to use.

**Reply by IMGROOT2** [reply]
completely useless and no real point anyway, LOL

**Reply by grimsteel** [reply]
> who gave you the money?

Ask Omkar/Nikhil in the discord server what day it is with regards to me

Also I have a nice email address: admin@vocabustudy.org (I didn't buy google workspace - this is just email forwarding)
Omkar wants "omkaradmin@vocabustudy.org" so ask me if you want one

**Reply by IMGROOT2** [reply]
i'd love one, lol. ruhan.admin@vocabustudy.org works haha!

should omkar's be omkar.admin or just omkaradmin?

**Reply by grimsteel** [reply]
> can i send an announcement?

Sure you can do it with Omkar so nobody complains about being left out.......... (trust me that has happened before)

**Reply by IMGROOT2** [reply]
> Sure you can do it with Omkar so nobody complains about being left out.......... (trust me that has happened before)

we did it together, LOL!

**Reply by PrakmO** [reply]
ok

**Reply by IMGROOT2** [reply]
https://knowt.io/ 

competition help

**Reply by grimsteel** [reply]
It's way better than Vocabustudy...

**Reply by PrakmO** [reply]
> It's way better than Vocabustudy...

But as I said in Skype, .org > .io so vocabustudy.org > knowt.io by transitive property

**Reply by IMGROOT2** [reply]
we need like some proprietary feature that we can patent or whatever

**Reply by PrakmO** [reply]
> we did it together, LOL!

like 10 seconds apart

**Reply by grimsteel** [reply]
> > It's way better than Vocabustudy...
> 
> But as I said in Skype, .org > .io so vocabustudy.org > knowt.io by transitive property

Transitive property of TLDs

**Reply by IMGROOT2** [reply]
ohhh so you guys use skype LOL

**Reply by IMGROOT2** [reply]
> > > It's way better than Vocabustudy...
> > 
> > 
> > But as I said in Skype, .org > .io so vocabustudy.org > knowt.io by transitive property
> 
> Transitive property of TLDs

ought to be in some sort of math textbook ngl

**Reply by IMGROOT2** [reply]
> > we did it together, LOL!
> 
> like 10 seconds apart

haha yeah

**Reply by grimsteel** [reply]
The fundamental property of History Tests should be in all history textbooks (I made that last year after realizing that teachers just copy questions off the internet)

**Reply by IMGROOT2** [reply]
teachers are as hardworking when teaching students as they are lazy when making lesson plans lol

**Reply by grimsteel** [reply]
Yes

**Reply by PrakmO** [reply]
> ohhh so you guys use skype LOL

Yes
Siddhant, Nikhil and I use Skype

**Reply by PrakmO** [reply]
Skype > Discord

**Reply by IMGROOT2** [reply]
lmao you really love inequalities don't you

**Reply by PrakmO** [reply]
yes I <3 inequalities
yes I is less than 3 inequalities

**Reply by IMGROOT2** [reply]
> > ohhh so you guys use skype LOL
> 
> Yes Siddhant, Nikhil and I use Skype

can I join your gc LOL

**Reply by ShreyanKhanna** [reply]
Yea same LMAO
